labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +101,14 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    var maxLength: Int = 0
+    for(line <- lines) {
+      for(string <- line.split(" ")) {
+        if(string.length > maxLength) {
+          maxLength = string.length
+        }
+      }
+    }
+    maxLength
```

Comment:
I think functional would have been the way to go here. Try using math and reduce!